sqlglot/expressions.py

```diff
@@ -3505,11 +3505,13 @@ class Matches(Func):
 
 
 class Max(AggFunc):
-    arg_types = {"this": True, "expression": False}
+    arg_types = {"this": True, "expressions": False}
+    is_var_len_args = True
 
 
 class Min(AggFunc):
-    arg_types = {"this": True, "expression": False}
+    arg_types = {"this": True, "expressions": False}
+    is_var_len_args = True
```

Comment:
This is probably a breaking change, let me know if I missed something by changing the `expression` arg. The `is_var_len_args` change was needed to support SQLite's variable length `MIN` and `MAX` variants.